docs/src/man/getting_started.md

```diff
@@ -26,6 +26,20 @@ Throughout the rest of this tutorial, we will assume that you have installed the
     See [here](https://jupyter-client.readthedocs.io/en/stable/kernels.html) for information about location
     and specification of Jupyter kernels.
 
+
+## DataFrame Introduction
+ 
+A DataFrame is a table that can contain data of various types side-by-side. It is a two-dimensional data structure with three main components: rows, columns, and the data:
+
+![DataFrame_Concept](https://user-images.githubusercontent.com/8378161/80563049-9ec88180-899e-11ea-87da-a850bd24bb5d.png)
```

Comment:
we keep extra data in docs/src/assets directory so that the repository is self contained.